src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -149,3 +214,41 @@
             :on-change      on-tab-change
             :data           tabs-data}]
           [tabs/view {:selected-tab @selected-tab}]]]))))
+
+(defn view
+  [_]
+  (let [{:keys [top]}              (safe-area/get-insets)
+        theme                      (quo.theme/use-theme)
+        title-bottom-coord         (rn/use-ref-atom 0)
+        set-title-bottom           (rn/use-callback
+                                    (fn [_ y _ height]
+                                      (reset! title-bottom-coord
+                                        (+ y height -100 (if platform/ios? (- top) top)))))
+        scroll-amount              (reanimated/use-shared-value 0)
+        title-opacity              (reanimated/use-shared-value 0)
+        collectible                (rf/sub [:wallet/last-collectible-details])
+        {:keys [preview-url
+                collection-data
+                collectible-data]} collectible
+        {preview-uri :uri}         preview-url
+        {collectible-name :name}   collectible-data
+        {collection-name :name}    collection-data]
+
+    (rn/use-unmount #(rf/dispatch [:wallet/clear-last-collectible-details]))
+
+    [rn/view {:style (style/background-color theme)}
+     [animated-header
+      {:scroll-amount scroll-amount
+       :title-opacity title-opacity
+       :page-nav-type :title-description
+       :picture       preview-uri
+       :title         collectible-name
+       :description   collection-name
+       :theme         theme}]
+     [reanimated/scroll-view
```

Comment:
hi, why this is reanimated view instead of simple view?

Reply:
Hi @Parveshdhull !

If you are referring to the `reanimated/scroll-view`, the reason is we  are capturing its `on-scroll` callback to animate a shared-value, since we wanted it to properly report the data, I thought it was better to use the reanimated one.

Reply:
hi Thank you for giving the context.
Most probably we will not get any benefit using reanimated here, unless we are using `animated-scroll-handler` something like https://github.com/status-im/status-mobile/blob/8f0840e913506c0c32157f403e72f5ecb80768b0/src/status_im/contexts/chat/messenger/messages/list/view.cljs#L418

Reply:
I mean there's a very common standard in react to add 'use-' as a prefix to any hook

Reply:
Thanks for the suggestion, I'll try removing it and using a normal scroll view to check if it behaves the same :+1: